### ISSUE TITLE
Use cosmwasm-check from CosmWasm 1.1

### ISSUE THREAD

**Issue by webmaster128** [neutron-org/neutron-contracts#28]
The code in https://github.com/neutron-org/neutron-contracts/tree/main/utils/check_contract is a copy of our check_contract example. This has been improved and published as a standalone tool called cosmwasm-check.

See
- https://crates.io/crates/cosmwasm-check
- https://github.com/CosmWasm/cosmwasm/issues/1371
- https://github.com/CosmWasm/cosmwasm/blob/main/CHANGELOG.md#110---2022-09-05

The tool allows you to provide custom [capabilities](https://github.com/CosmWasm/cosmwasm/blob/main/docs/CAPABILITIES.md) using like this:

```
cosmwasm-check --available-capabilities iterator,osmosis,friendship artifacts/hackatom.wasm
```

You can now also pass an arbitrary number of wasm files to it as positional arguments:

```
cosmwasm-check --available-capabilities iterator,osmosis,friendship artifacts/*.wasm
```

Is there anything else you need to customize?

**Comment by quasisamurai** [comment]
I appreciate both your work and your issue opening initiative. Thanks, tool works awesome

**Comment by foxpy** [comment]
Fixed by #32
